pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -18,7 +18,6 @@ import (
 	nsutils "github.com/rancher/rancher/pkg/namespace"
 	pkgrbac "github.com/rancher/rancher/pkg/rbac"
 	"github.com/rancher/rancher/pkg/types/config"
-	"github.com/rancher/wrangler/pkg/relatedresource"
```

Comment:
Can you remove this import? I see it still being used on L135 (unlike in the 2.6 branch).